### DISCUSSION TITLE
Is it possible to refine camera intrinsics for one camera only?

### DISCUSSION THREAD

**Discussion by tworkool** [colmap/colmap#2607] [Q&A]
Hello,
I am doing a project for my masters thesis using HLoc, which uses COLMAP as the underlying SfM pipeline. I have a use case where I know the camera intrinsics of all cameras except one. I would like to refine only the camera intrinsics of that one camera with unknown intrinsics. Is this possible in COLMAP?

**Comment by jared-krauss** [comment]
Did you ever find an answer to this?
